src/u2f.c

```diff
@@ -103,6 +104,26 @@ typedef struct __attribute__((__packed__)) {
 
 #pragma GCC diagnostic pop
 
+static component_t* _create_refresh_webpage(void)
+{
+    return info_centered_create("Refresh webpage", NULL);
+}
+
+static component_t* _nudge_label = NULL;
+
+static void _nudge_init_cb(component_t* component)
```

Comment:
slightly unfortunate name, it sounds like this is called when the nudge is initialized :sweat_smile: no big deal though

Reply:
fixed, I think `_nudge_label_cb` is a better name.